files/static/style.css

```diff
@@ -0,0 +1,234 @@
+html, body {
+  width: 100%;
+}
+
+body {
+  margin: 0;
+  background-color: #f5f5f5;
+  font-family: Helvetica, "Roboto", Tahoma, sans-serif;
```

Comment:
Hmm, this originally comes from some Material design templates. I'll move Robot first here.